### ISSUE TITLE
Unpack with using recovery volume fail when path or rar file was not given

### ISSUE THREAD

**Issue by mplpl** [mplpl/unrar_amiga#29]
RAR can extract a multivolume archive when a volume is missing if recovery volume was created (when packing). But on MorphOS, AmigaOS4 and AROS it does not work if rar file was given to unrar without a path:

```
System:Data/c/new_tests> list
unrar_test_57.part1.rev           2000 ----rwed 14-Feb-20 20:33:00
unrar_test_57.part3.rar           1429 ----rwed 11-Feb-20 20:46:00
unrar_test_57.part1.rar           2000 ----rwed 11-Feb-20 20:46:00
3 files - 5429 bytes used

System:Data/c/new_tests> /unrar_amiga/unrar x unrar_test_57.part1.rar
UNRAR 5.80 freeware      Copyright (c) 1993-2019 Alexander Roshal

MorphOS port by Marcin Labenski

Extracting from unrar_test_57.part1.rar
Extracting  UnRAR-5.80.readme                                         96%
Cannot find volume unrar_test_57.part2.rar
UnRAR-5.80.readme    - checksum error
Total errors: 1
```

However, if rar was passed with a path it works fine:

```
System:Data/c/new_tests> /unrar_amiga/unrar x /new_tests/unrar_test_57.part1.rar
UNRAR 5.80 freeware      Copyright (c) 1993-2019 Alexander Roshal

MorphOS port by Marcin Labenski

Extracting from /new_tests/unrar_test_57.part1.rar
Extracting  UnRAR-5.80.readme                                         96%
Calculating checksums of all volumes.
/new_tests/unrar_test_57.part1.rev
1 recovery volumes found
/new_tests/unrar_test_57.part1.rar
Cannot find volume /new_tests/unrar_test_57.part2.rar
/new_tests/unrar_test_57.part3.rar
1 volumes missing
Reconstructing... 100%
Extracting from /new_tests/unrar_test_57.part2.rar
...         UnRAR-5.80.readme
Extracting from /new_tests/unrar_test_57.part3.rar
...         UnRAR-5.80.readme                                         OK
All OK
```

On AmigaOS3 it works fine in both cases.

**Comment by mplpl** [comment]
This issue is caused because in order to find 'rev' file with matching name, unrar is calling opendir function with directory in which given rar file is located. When rar was passed to unrar command without a path (just file name), the dir is set to ".". But unfortunately Amiga version of opendir (newlib/libnix) does  not treat "." as a current directory. 
And on AmigaOS3, unrar uses ixemul that actually accepts "." as a current dir - that is why this error is not present when running unrar on AmigaOS3.

In order to fix, I will just pass "" to opendir instead of "." - it works fine in all Amiga implementations.

**Comment by mplpl** [comment]
Resolved by fe2743a1488f84b88e3f82c0477a6e0901922a12
